### DISCUSSION TITLE
Can I install Docusaurus on my cpanel?

### DISCUSSION THREAD

**Discussion by umairtipu** [facebook/docusaurus#7719] [Q&A]
Hi everyone, I have installed ngnix on my cpanel and there is wordpress website running on the server. I have to install docusaurus project and created a folder there and install a basic docusaurs application in that folder. The problem is when i build the application, only the home page works fine and when I clicks on the link it is showing the link as the part of wordpress website and showing a very basic markdown file. Can anyone help me how can I fix the routing issue?

**Comment by slorber** [comment]
It's impossible to understand your problem with so few information.

Please give us the URL of your website, or at the very least a screenshot demonstrating the problem.

Don't assume we know what cpanel is, because we don't 😅

**Reply by onyxdevs** [reply]
cpanel is a hosting management control panel 😅, it's famous across agencies but usually it doesn't have Node installed.

**Comment by pgr224** [comment]
I think you need to modify htaccess file also
